modules/aws/mongo_atlas/main.tf

```diff
@@ -0,0 +1,44 @@
+data "aws_caller_identity" "account" {}
```

Comment:
This folder should be `mongo_atlas_vpc_peering` instead.

Reply:
Updated foldername to mongo_atlas_vpc_peering